### DISCUSSION TITLE
intermediate snapshots and space usage?

### DISCUSSION THREAD

**Discussion by silenius** [zrepl/zrepl#671] [Q&A]
Hello,

I'm busy to setup (and test) zrepl here at $work and it works well, one thing I'm concerned about is that intermediate snapshots might just take up more and more space if zrepl doesn't clone and promote some snapshot at some point (the last that respects the retention period) ..?

Julien

**Comment by problame** [comment]
Sorry for the late response.

> if zrepl doesn't clone and promote some snapshot at some point (the last that respects the retention period) ..?

And sorry again, I don't get your concern.

Is your problem that zrepl is keeping to many snapshots? Then adjust the pruning rules accordingly, so that you don't keep the snapshots for as long.

If that's not your problem, please explain what you mean by `clone` and `promote`, maybe using an example.
